README.rst

```diff
@@ -19,13 +23,32 @@ Updating SQLite Source Dependencies
 
       git checkout -b <branch-name>-sqlite upstream/sqlite
 
-3. Download SQLite source from ``sqlite.org``.
+3. Download SQLite source from `sqlite.org <https://www.sqlite.org>`_.
 
 4. Unzip it to the checked out branch on step 2.
 
 5. Commit and push the changes.
 
 6. Create the PR, with ``sqlite`` as the base branch.
 
-Once the PR has been merged, tag the commit as ``sqlite-<full version specified in CPython's PCbuild/get_externals.bat>``.
+Once the PR has been merged, tag the commit as ``sqlite-<full version to be used in CPython's PCbuild/get_externals.bat>``.
+
+For updating ``sqlite``, ``bzip2``, ``xz``, and ``zlib``, follow the above
+instructions.
+
+For ``tcl`` and ``tk``, after step 5, `cpython-bin-deps
+<https://github.com/python/cpython-bin-deps/>`_
+needs to be updated.
+
+For ``openssl``, step 5 will be more complicated :)
```

Comment:
For each of these, we can link to the source site:

sqlite: https://www.sqlite.org/download.html
bzip2: http://www.bzip.org/downloads.html
xz: https://tukaani.org/xz/
zlib: https://zlib.net/
tcl/tk: https://tcl.tk/software/tcltk/download.html
openssl: https://www.openssl.org/source/

Reply:
Thanks, I added a "Download links" section listing the above.